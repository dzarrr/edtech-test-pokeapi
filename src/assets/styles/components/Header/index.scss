.header {
  background-color: #ff6961;
  padding: .5em;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
  
  .title{
    width: 100%;
    max-width: 480px;
    margin: 0 auto;
  }
}