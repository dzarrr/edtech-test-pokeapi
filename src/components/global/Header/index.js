const Header = ({ children }) => {
  return (
    <div className='header'>
      <div className="title">
        <h1>
          {children}
        </h1>
      </div>
    </div>
  )
}

export default Header