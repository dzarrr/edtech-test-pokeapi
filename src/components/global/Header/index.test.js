import React from 'react'
import { render } from '@testing-library/react'
import Header from 'components/global/Header'

test('renders element correctly', () => {
  const { asFragment } = render(<Header />)

  expect(asFragment()).toMatchSnapshot()
});
