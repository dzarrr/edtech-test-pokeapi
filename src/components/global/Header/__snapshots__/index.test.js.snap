// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders element correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <div
      class="title"
    >
      <h1 />
    </div>
  </div>
</DocumentFragment>
`;
